app/lib/preprocessor.js

```diff
@@ -35,39 +35,46 @@ module.exports = function(options, specData) {
           delete path[method]
           return
         }
-        var operation = path[method]
-        operation.path = pathName
-        operation.method = method
-        // Draw links from tags to operations referencing them
-        var operationTags = operation.tags || ['default']
-        operationTags.forEach(function(tag) {
-          if (!tagsByName[tag]) {
-            // New implicit declaration of tag not defined in global "tags"-object
-            // https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#user-content-swaggerTags
-            var tagDefinition = {
-              name: tag,
-              operations: []
+        // operations can be a single or an array
+        var operations = path[method]
+        if (!_.isArray(operations)) {
+          operations = [operations];
+        }
+        operations.forEach(operation => {
+          operation.path = pathName;
+          operation.method = method;
+          // Draw links from tags to operations referencing them
+          var operationTags = operation.tags || ['default'];
+          operationTags.forEach(function(tag) {
+            var tagDefinition = tagsByName[tag];
+            if (!tagDefinition) {
+              // New implicit declaration of tag not defined in global "tags"-object
+              // https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#user-content-swaggerTags
+              tagDefinition = {
+                name: tag,
+                operations: []
+              };
+              tagsByName[tag] = tagDefinition;
+              copy.tags.push(tagDefinition);
             }
-            tagsByName[tag] = tagDefinition
-            copy.tags.push(tagDefinition)
-          }
-          if (tagsByName[tag]) {
-            tagsByName[tag].operations = tagsByName[tag].operations || []
-            tagsByName[tag].operations.push(operation)
-          }
-        })
-        // Join parameters with path-parameters
-        operation.parameters = (operation.parameters || [])
-          .concat(pathParameters)
-          .filter(function(param) {
-            if (param.in === 'body') {
-              operation._request_body = param
-              return false
+            if (!tagDefinition.operations) {
+              tagDefinition.operations = [];
             }
-            return true
-          })
-        // Show body section, if either a body-parameter or a consumes-property is present.
-        operation._show_requst_body_section = !!operation._request_body //|| operation.consumes
+            tagDefinition.operations.push(operation);
```

Comment:
was this a manual change? I wonder if we just merge with the latest version of the library master we'll also get this for free? https://github.com/lilt/spectacle/blob/master/app/lib/preprocessor.js#L56

Reply:
I just refactored this and I think the diff is a bit confusing here. previously it was this line
```javascript
          if (tagsByName[tag]) {
            tagsByName[tag].operations = tagsByName[tag].operations || []
            tagsByName[tag].operations.push(operation)
          }
```

Reply:
just checked the original library master and there isn't anything new that's interesting. not many commits either